### ISSUE TITLE
Add co-collectors to elasticsearch model and reconciliation endpoint

### ISSUE THREAD

**Issue by dshorthouse** [bionomia/bionomia#57]
As in the subject. Part of solution will be to assess in what form the co-collector strings should take. Parsed and atomized? Unparsed as simple arrays?

**Comment by JCGiron** [comment]
It would be nice if you could assign co-collector specimens while assigning another collector's specimens.
For example:
![image](https://user-images.githubusercontent.com/37313282/97811106-f7265780-1c3d-11eb-81bb-ff35a429436e.png)
If I was assigning specimens for Ligia R. Benavides here, it would be nice if I could somehow assign specimens to Gonzalo Giribet, perhaps if the co-collector is already in the system, with a click you could open a new window with their specimens? or a new window for the specimen to assign all the agents?

**Comment by dshorthouse** [comment]
@JCGiron Agreed! There are many little instruments that must be aligned into a little concert to do this well, but we're getting closer to something like this. This original ticket was going about it from the other direction, i.e. when both Ligia & Gonzalo have both been independently attributed these same specimens, we can use this for other specimens that have not yet been attributed.

There are a number of questions that come into play to make a quick user interface that combine a few things:
- is the person known to Bionomia? (i.e. exists in ORCID or Wikidata)
- does the agent namestring correspond to a person known to Bionomia? (a search problem)
- has the attribution already been made?

**Comment by dshorthouse** [comment]
Co-collectors added to ES model but not yet added/used in reconciliation end-point.
